ipsec_offload_plugin/combined_rcp_ipsec_offload.conf

```diff
@@ -0,0 +1,26 @@
+# conf file for ipsec_offlaod plugin
```

Comment:
There must be some code change which reads this new file instead of old one??

Reply:
Good catch. I was planning to update documentation to copy this conf file in /usr/share/stratum location as ipsec_offload.conf. Is this okay or should we change the code to read this file instead?

Reply:
I have added doc update as part of this PR

Reply:
That file need to be renamed as ipsec_offload.conf ?? Or as per the current code you need add a ifdef to read separate files when separate p4 is in use!!!